lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -2778,7 +2778,7 @@ public function getListNamespacesSQL()
     }
 
     /**
-     * @param string $database
+     * @param string|null $database
```

Comment:
Why null? None of the implementations expect null.

Reply:
```
ERROR: NullArgument - tests/Doctrine/Tests/DBAL/Platforms/SQLAnywhere12PlatformTest.php:55:50 - Argument 1 of Doctrine\DBAL\Platforms\SQLAnywhere12Platform::getListSequencesSQL cannot be null, null value provided to parameter with type string (see https://psalm.dev/057)
            $this->platform->getListSequencesSQL(null)
```


Looks like SQLAnywhere12 (at least) does https://github.com/doctrine/dbal/blob/155d028be084ef69110d09938c57c351a2126e5d/lib/Doctrine/DBAL/Platforms/SQLAnywhere12Platform.php#L65-L68

Reply:
Those methods ignore the passed value. It doesn't mean that they accept `NULL`.

Reply:
Ok, I will use `''` in tests that use `null` here.

Reply:
If that feels wrong, maybe we should use `?string $database` when merging up.

Reply:
`''` is better, it's at least of the correct type. `?string $database` is not an option. None of the implementations accept an optional DB name in `master` and this is done deliberately.